src/lib/gui/gl_2d_display.h

```diff
@@ -15,18 +15,20 @@ class gl_2d_display
 {
 protected:
 	static constexpr GLfloat PLANE[] = {
-		-1.0f, -1.0f, 0.0f,
-		 1.0f, -1.0f, 0.0f,
-		-1.0f,  1.0f, 0.0f,
-		 1.0f, -1.0f, 0.0f,
-		 1.0f,  1.0f, 0.0f,
-		-1.0f,  1.0f, 0.0f
+	    -1.0f, -1.0f, 0.0f,
+	     1.0f, -1.0f, 0.0f,
+	    -1.0f,  1.0f, 0.0f,
+	     1.0f, -1.0f, 0.0f,
+	     1.0f,  1.0f, 0.0f,
+	    -1.0f,  1.0f, 0.0f
 	};
 
 public:
 	gl_2d_display()
-		: _p(create())
+	    : _p(create())
 	{
+		glGenBuffers(3, _vbo.data());
```

Comment:
Creating our VBOs (and more than one of them) in advance. I don't really know what I'm doing here, but this did seem to help.